src/rSW_Site.c

```diff
@@ -184,7 +185,7 @@ static SEXP onGet_SW_SWRCp(void) {
 	int i, k;
 	SW_SITE *v = &SoilWatRun.Site;
 	SEXP SWRCp, SWRCp_names, SWRCp_names_y;
-	RealD *p_SWRCp;
+	double *p_SWRCp;

```

Comment:
onGet_SW_SWRCp() should now
- copy `v->soils.swrcpMineralSoil[i][k]` into "p_SWRCp"
- copy `v->swrcpOM` into new "omSWRCp"

This is because SOILWAT2 now distinguishes between SWRCp in the mineral soil `swrcpMineralSoil` (user inputs), SWRCp in the organic matter `swrcpOM` (user input) and the calculated bulk soil SWRCp `swrcp`. For rSOILWAT2, `SWRCp` (slot "SWRCp" of S4 class "swSoils") is equivalent to SWRCp in the mineral soil (user inputs) -- but we cannot rename it for backwards compatibility .... That also means that we need now to add a new slot "omSWRCp" to S4 class "swSoils" for equivalency with SOILWAT2. And similarly, S4 class "swSite" gains a new slot "depthSapric"